ultravox_client/src/main/java/ai/ultravox/UltravoxSession.kt

```diff
@@ -24,6 +27,35 @@ class UltravoxSession(
     private val state = UltravoxSessionState()
     private var socket: WebSocket? = null
     private var room: Room = LiveKit.create(ctx)
+    private var speakerMuted: Boolean = false
+
+    val isMicMuted: Boolean
+        get() = !room.localParticipant.isMicrophoneEnabled()
+    
+    suspend fun setMicMuted(value: Boolean) {
+        room.localParticipant.setMicrophoneEnabled(!value)
```

Comment:
Does setMicrophoneEnabled require suspension?  If not, I think you can simplify this by changing isMicMuted to just micMuted above and making this its setter.  Then you can update toggleMicMuted also.